e2-studio-star-rx72n-firmware/src/boot/r_bsp/r_rx_intrinsic_functions.c

```
@@ -0,0 +1,1237 @@
+* Function Name: R_BSP_MulAndAccOperation_L
+* Description  : Performs a multiply-and-accumulate operation with the initial value specified by init, the number of
+*                multiply-and-accumulate operations specified by count, and the start addresses of values to be 
+*                multiplied specified by addr1 and addr2.
+* Arguments    : init   - Initial value.
+*                count  - Count of multiply-and-accumulate operations.
+*                *addr1 - Start address of values 1 to be multiplied.
+*                *addr2 - Start address of values 2 to be multiplied.
+* Return Value : result - Lower 64 bits of the init + S(data1[n] * data2[n]) result. (n=0, 1, ..., const-1)
+***********************************************************************************************************************/
+#if defined(__GNUC__)
+long long R_BSP_MulAndAccOperation_L(long long init, unsigned long count, long* addr1, long* addr2)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'R_BSP_MulAndAccOperation_L' of convertible types are easily swapped by mistake

(bugprone-easily-swappable-parameters)